pkg/sdk/sdk.go

```diff
@@ -1285,13 +1285,6 @@ type SDK interface {
 	//  fmt.Println(err)
 	SendInvitation(ctx context.Context, invitation Invitation, token string) (err error)
```

Comment:
Can we have like function like `SendDomainInvitation`, `ListDomainInvitation` , `DeleteDomainInvitation` , these function are used by domain admins 

And to list user's invitaiton we have function like 
`ListInvitations`, `AcceptInvitations`, `RejectInvitations`